magicblock-committor-service/src/tasks/mod.rs

```diff
@@ -102,6 +109,116 @@ pub struct CommitTask {
     pub commit_id: u64,
     pub allow_undelegation: bool,
     pub committed_account: CommittedAccount,
+    base_account: Option<Account>,
```

Comment:
`CommitTask` maybe ambiguos name now since when we were implementing this, we didn't think about another commit option via diffs. Since task is representation of instruction that is performed on chain we need to add new type of task for diffs. `CommitWithDiffTask`.
Please also rename CommitTask -> `CommitDataTask` or something like that to highlight that it commit of the whole account state and not the diff